ntbs-service/Models/Notification.cs

```diff
@@ -1,18 +1,127 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ntbs_service.Models.Enums;
 
 namespace ntbs_service.Models
 {
     public class Notification
     {
         public int NotificationId { get; set; }
-
         public virtual PatientDetails PatientDetails { get; set; }
         public virtual ClinicalDetails ClinicalDetails { get; set; }
-
         public virtual List<NotificationSite> NotificationSites { get; set; }
         public virtual Episode Episode { get; set; }
         public virtual PatientTBHistory PatientTBHistory { get; set; }
         public virtual ContactTracing ContactTracing { get; set; }
         public virtual SocialRiskFactors SocialRiskFactors { get; set; }
+
+        public string FullName {
```

Comment:
I assume EF doesn't complain about these as they don't have setters, so it doesn't try to add them to the model?

Reply:
Oh good point!
This would suggest so: https://docs.microsoft.com/en-us/ef/core/modeling/included-properties#conventions
Worth checking though!
But if we needto exclude them, then: https://www.learnentityframeworkcore.com/configuration/fluent-api/ignore-method#excluding-a-property-from-mapping